### ISSUE TITLE
Request : Microsoft Edge compatibility

### ISSUE THREAD

**Issue by Matt-One** [joncage/ed-scout#184]
Hi and thank you for the program u make.
Is it possible to make it compatible with the new microsoft edge? It is always based on Chromium like google chrome.

Thank you :)

**Comment by joncage** [comment]
Have you tried running EDScout headless using command line options and conneting a browser to it?

Some instructions here: https://github.com/joncage/ed-scout#command-line-options

**Comment by Matt-One** [comment]
> Have you tried running EDScout headless using command line options and conneting a browser to it?
> 
> Some instructions here: https://github.com/joncage/ed-scout#command-line-options

thank you for your reply, i haven't tried it because i don't know how to connect another browser using command lines, the only command line i know is "EDScout.exe -port 12345 -host 192.168.1.31" as you wrote in your readme. Any tip? :) thank you

**Comment by joncage** [comment]
Yep. Enter the name of your computer or its up address for the `-host` part of the command line. 

Then in your browser go to `http://<IP or host name you ran it with>:<port>`

**Comment by Matt-One** [comment]
> Yep. Enter the name of your computer or its up address for the `-host` part of the command line.
> 
> Then in your browser go to `http://<IP or host name you ran it with>:<port>`

Doing that i always need Chrome to be installed?

**Comment by joncage** [comment]
I've never tried. If it doesn't work like that already, I'm sure I could add a command line option for it to run headless/chromeless.

**Comment by Matt-One** [comment]
> I've never tried. If it doesn't work like that already, I'm sure I could add a command line option for it to run headless/chromeless.

i deleted Chrome and tried to start Edscout, the log says :

```
2021-04-19T12:44:38.724Z - EDScoutWebUI-EDScout - INFO - ED Scout v1.8.2 Starting
2021-04-19T12:44:38.860Z - EDScoutWebUI-EDScout - INFO - Found existing Scout instance PID:4344, killing off before continuing
2021-04-19T12:44:38.939Z - flaskwebgui-flaskwebgui - ERROR - [WinError 2] Impossibile trovare il file specificato
NoneType: None
2021-04-19T12:44:38.939Z - flaskwebgui-flaskwebgui - ERROR - Failed to detect chrome location from registry
2021-04-19T12:44:38.940Z - EDScoutWebUI-EDScout - INFO - Looking for HUD overrides in C:\Users\Matt-One\AppData\Local\Frontier Developments\Elite Dangerous\Options\Graphics\GraphicsConfigurationOverride.xml
2021-04-19T12:44:38.942Z - EDScoutWebUI-EDScout - INFO - Successfully remapped styles to match Elite HUD
2021-04-19T12:44:58.248Z - VersionChecker-VersionChecker - INFO - Version check: Up to date
2021-04-19T12:44:58.249Z - EDScoutWebUI-EDScout - INFO - Background thread launched and awaiting data..
2021-04-19T12:45:59.082Z - JournalInterface-JournalInterface - INFO - Journal created: C:\Users\Matt-One\Saved Games\Frontier Developments\Elite Dangerous\Journal.210419144559.01.log
```
but the process is in memory and i can access edscout through Microsoft Edge with my local IP, it would be great if i can access edscout if a separate windows like if Chrome were installed, but anyway it work :)

OT: It's normal that i see all the line in Grey and not in Orange like your screenshot in the homepage?

**Comment by joncage** [comment]
Grey indicates a system or body has already been discovered. Head a bit further out and you'll see more orange 👌🏻

**Comment by Matt-One** [comment]
> Grey indicates a system or body has already been discovered. Head a bit further out and you'll see more orange 👌🏻

Thank you so much for your help :)
